mappings/net/minecraft/world/gen/feature/ConfiguredFeatures.mapping

```diff
@@ -1,4 +1,116 @@
 CLASS net/minecraft/class_5464 net/minecraft/world/gen/feature/ConfiguredFeatures
+	FIELD field_26090 FOREST_FLOWER_VEGETATION_CONFIG Lcom/google/common/collect/ImmutableList;
```

Comment:
```suggestion
	FIELD field_26090 FOREST_FLOWER_VEGETATION_CONFIGS Lcom/google/common/collect/ImmutableList;
```
i'd guess this is configs as this is a list

Reply:
```suggestion
	FIELD field_26090 FLOWER_FOREST_VEGETATION_CONFIGS Lcom/google/common/collect/ImmutableList;
```